app/php/composer.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+from app.apache.apache import Apache
+from app.apache.proxyPass import ProxyPass
+
+
+class Composer:
+
+    def run(self, serverName=None, port=None):
+        os.system('composer install')
+        os.system('php artisan key:generate')
+        os.system('php artisan migrate')
+        os.system('php artisan db:seed')
```

Comment:
Can you make a seperate method for this? Cause every laravel project doesn't need to have a seed. So make a new function that does this.